src/pkg/selectors/groups.go

```diff
@@ -309,7 +310,76 @@ func (s *groups) LibraryItems(libraries, items []string, opts ...option) []Group
 // -------------------
 // ItemInfo Factories
 
-// TODO
+// MessageCreator produces one or more groups channelMessage info scopes.
+// Matches any channel message created by the specified user.
+// If any slice contains selectors.Any, that slice is reduced to [selectors.Any]
+// If any slice contains selectors.None, that slice is reduced to [selectors.None]
+// If any slice is empty, it defaults to [selectors.None]
+func (s *GroupsRestore) MessageCreator(creator string) []GroupsScope {
```

Comment:
Following from https://github.com/alcionai/corso/pull/4092#discussion_r1302921568 , do we need separate creator and owner?

Reply:
See reply to that comment.  I think I'm going to keep with the separation, unless there's strong feelings otherwise.